logicrunner/builtin/foundation/helpers.go

```diff
@@ -45,3 +46,18 @@ func GetRequestReference() insolar.Reference {
 func GetObject(ref insolar.Reference) ProxyInterface {
 	panic("not implemented")
 }
+
+// Get reference CostCenter contract.
+func GetCostCenter() insolar.Reference {
+	return genesisrefs.ContractCostCenter
+}
+
+// Get reference MigrationAdminMember contract.
+func GetMigrationAdminMember() insolar.Reference {
```

Comment:
Where is []MigrationDaemonsRef ?

Reply:
this next cr https://insolar.atlassian.net/browse/WLT-808